src/view/renderer.js

```diff
@@ -580,6 +582,17 @@ export default class Renderer {
 			return;
 		}
 
+		if ( oldViewSelection && this._areSimilarSelections( oldViewSelection, this.selection ) ) {
+			const data = {
+				oldSelection: oldViewSelection,
+				currentSelection: this.selection
+			};
+
+			log.warn( 'renderer-selection-similar: Rendering selection was skipped due to its similarity to the previous selection.', data );
```

Comment:
It's super verbose now... but perhaps it's better to keep it logged like this for a time being so we know what's going on.

However, let's improve the text:

`renderer-skipped-selection-rendering: The selection was not rendered due to its similarity to the current one.`